src/Validator.js

```diff
@@ -0,0 +1,37 @@
+import CustomError from './Error.js';
+import ERROR_MESSAGES from './constants/error.js';
+import { MAX_NAME_LENGTH, NAME_SEPARATOR } from './constants/index.js';
+
+class Validator {
+  static validateRacers(userInput) {
+    this.validateEmptyUserInput(userInput);
+
+    if (userInput.split(NAME_SEPARATOR).length < 2) {
+      throw new CustomError(ERROR_MESSAGES.INVALIDATE_SEPARATOR);
+    }
+  }
+
+  static validateTotalRound(userInput) {
+    this.validateEmptyUserInput(userInput);
+
+    const totalRound = Number(userInput);
+
+    if (!totalRound || totalRound < 0 || !Number.isInteger(totalRound)) {
+      throw new CustomError(ERROR_MESSAGES.INVALIDATE_TOTAL_ROUND);
+    }
```

Comment:
```suggestion
    if (Number.isNaN(totalRound) || totalRound <= 0 || !Number.isInteger(totalRound)) {
      throw new CustomError(ERROR_MESSAGES.INVALIDATE_TOTAL_ROUND);
    }
```
시도 횟수를 검증하는 코드에서 !totalRound를 사용하는 것은 다소 좋지 않은 방식이라고 생각됩니다. Number.isNaN을 사용하여 totalRound가 숫자가 아닌 경우를 명확히 검증하는 것이 더 명확하다고 생각됩니다.😀😀😀